src/SFA.DAS.ContentApi.UnitTests/AutoFixture/ProviderDbCustomizations.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using AutoFixture;
+using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Diagnostics;
+using SFA.DAS.ContentApi.Data;
+
+namespace SFA.DAS.ContentApi.UnitTests.AutoFixture
+{
+    public class ProviderDbCustomizations : ICustomization
```

Comment:
Rename to "ContentDbCustomizations" ? same goes for a couple of the other Autofixture helper classes